package.json

```diff
@@ -68,66 +68,66 @@
     "@babel/plugin-syntax-import-meta": "7.10.4",
     "@babel/plugin-transform-async-to-generator": "7.13.0",
     "@babel/plugin-transform-classes": "7.13.0",
-    "@babel/plugin-transform-runtime": "7.13.9",
+    "@babel/plugin-transform-runtime": "7.13.10",
     "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "7.13.9",
+    "@babel/preset-env": "7.13.10",
     "@babel/preset-typescript": "7.13.0",
     "@babel/register": "7.13.8",
-    "@babel/runtime": "7.13.9",
-    "@commitlint/cli": "8.3.5",
-    "@commitlint/config-conventional": "8.3.4",
-    "@octokit/rest": "16.43.2",
-    "@types/async": "3.2.4",
+    "@babel/runtime": "7.13.10",
+    "@commitlint/cli": "12.0.1",
```

Comment:
Update major packages is not a good idea. I'd go step by step, but not all in once PR, otherwise you will have a lot of o fun fixing stuff. Also remember there is still Node.js 8 compatibility provided, even if the CI is not running on Node 8 anymore.

Reply:
What does that pdf mean? I never download or open files. I'd recommend writing all text you need on the PR body, is more accessible to everyone, from any device.